### ISSUE TITLE
Please add Portuguese Language to Crowdin

### ISSUE THREAD

**Issue by JANogueira** [modxcms/revolution#14659]
Hi,

Please add Portuguese language to Crowdin so I can help with Portuguese translations.
https://crowdin.com/project/modx-revolution

Thanks.

**Comment by Mark-H** [comment]
Would it make sense to complete "Portuguese (Brazilian)" rather than add a new language, or is that too different from "Portuguese"? https://crowdin.com/project/modx-revolution/pt-BR

**Comment by JANogueira** [comment]
Hi Mark,

Just for clarification, there is an "orthographic agreement" for all countries with Portuguese as primary native language that is implemented for several years now.
The objective of this agreement is to have all countries writing on the same way.
So I believe that there is little sense to talk about Portuguese from Portugal or Portuguese from Brazil (or Angola, or Mozambique, etc).

Please consider changing this to "Portuguese" only.
I am also willing to help with this translation.
Portuguese is my native language.

So this Portuguese translation will be the same for pt-pt and pt-br lexicons.

I will try to be as generic as possible trying to avoid expressions that can be typically from Portugal as there are some relevant semantic differences from Portugal and Brazil (as an example).
But I can check and continue the Portuguese translation that is already visible in Croudin.

Thanks

**Comment by Mark-H** [comment]
Gotcha. It does make more sense to switch `pt-br` to simply `pt` then. Not sure why it's like this to begin with.

**Comment by JANogueira** [comment]
Yes, indeed. Maybe we will have to wait for someone to change it in Crowdin.
Anyway I will use the current portuguese-brazilian one.

**Comment by Ibochkarev** [comment]
Made moving `pt-br` to `pt` in https://github.com/modxcms/revolution/commit/415ea252ef2b90ef6fbff51a53870bea604ac17e 

@modxbot close

**Comment by Blackspirits** [comment]
Hi,

but I don't agree with anything you've done because Portuguese in Portugal and Portuguese in Brazil are very different. The difference between the two is many times worse than British and American English.
There are hundreds of words that are spelled differently and others that are spelled the same but have different meanings.

Ex:
Utilizadores -> Usuário
Registo -> Registro
Comboio -> Trem
Chavena -> Xicara
etc,
These are just 4 examples, but I could be here all day listing completely different words.

The difference between the Portuguese of Portugal, Angola and Mozambique is minimal, for various reasons, one of which is that they were Portuguese colonies for longer and even more recently, while Brazil was independent 100 years before these countries and has undergone many changes due to immigration and influences from other countries such as France and Italy.

As for the "orthographic agreement" that the user mentioned, Brazil abandoned it a few years ago because it doesn't agree with it.
I'll give you another example: if a Portuguese person goes to Brazil, depending on the city they won't understand most things, and when Brazilians come to Portugal, they have a hard time understanding at first too.

For these and many other reasons, most websites, apps, even streaming services like Netflix, Disney+, etc., add pt-PT and pt-BR.

That's why I advise you to change "pt" to "pt-PT" and add "pt-BR"

Best Regards,
BlackSpirits

**Comment by JANogueira** [comment]
Indeed.
Many things changed over the recent past years.
I agree with the proposal to have pt-pt and pt-br.

**Comment by pbowyer** [comment]
As someone who started learning (european) Portuguese recently, I 💯 everything @Blackspirits has said.

**Comment by Ibochkarev** [comment]
https://crowdin.com/project/modx-revolution/pt-PT - available for translation

**Comment by Blackspirits** [comment]
Thanks for making Portuguese (Portugal) (`pt-PT`) available on Crowdin.

I reviewed the current repository/Crowdin setup and prepared draft PR #17009 to separate the `pt-PT` and `pt-BR` export paths properly.

The current generic `pt` lexicons are historically based on the former Brazilian Portuguese (`pt-br`) lexicons, which were moved to `pt` in 2019. Now that both regional variants are available, the current `%two_letters_code%` export pattern would send both `pt-PT` and `pt-BR` to the same `pt` paths.

The draft PR therefore adds explicit Crowdin mappings for `pt-PT` and `pt-BR`, while deliberately leaving the existing `pt` files untouched for backward compatibility. It also avoids editing any non-English lexicon files directly, so the actual translations can continue to be managed through Crowdin as required by the MODX contribution guidelines.

I have kept the PR in draft for now because the Crowdin export behaviour still needs to be validated through the project integration.

I am a Portuguese translator and I would be happy to take responsibility for the European Portuguese translation. If possible, I would also be interested in becoming a `pt-PT` reviewer/proofreader so I can help keep terminology and translations concise, natural, and consistent across MODX.
